src/three/store/ModelsStore.js

```diff
@@ -61,6 +62,17 @@ class ModelsStore {
     this.updateModelsSubject$.next(this.models);
     this.removeModelsSubject$.next(model);
   }
+
+  changeName(model, newName) {
+    const modelIndex = this.models.indexOf(model);
+    const matchModel = this.models[modelIndex];
+
+    if (modelIndex !== -1 && matchModel) {
+      matchModel.mesh.name = newName;
```

Comment:
Could you add setter and getter methods to ModelBase class and use it here?

Reply:
*also I'm wandering if we can re-implement Models classes to make them more reactive, I mean to connect them with rxjs. I'm not sure how exactly it will affect application performance and structure but I think that would make sense to separate operations on models list and single model. First type of operation will be cover by Store (in eg. add, remove) and the second by Model object itself (in eg. changeName, setIsInteractive) (this is only suggestion to consider)

Reply:
Updated